src/py42/clients/alerts.py

```diff
@@ -80,3 +80,12 @@ def update_state(self, status, alert_ids, note=None):
             :class:`py42.response.Py42Response`
         """
         return self._alert_service.update_state(status, alert_ids, note=note)
+
+    def update_note(self, alert_id, note):
+        """Add/update  a note to an alert.
```

Comment:
There is an extra space between `Add/update` and `a note...`

Reply:
done.